### ISSUE TITLE
🐋 sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: ui_settings

### ISSUE THREAD

**Issue by dbuttric** [automatic-ripping-machine/automatic-ripping-machine#829]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

The Docker image will not start successfully because of this error.

What can I do to troubleshoot this?

### To Reproduce

run the start_arm_container.sh

docker run -d \
    -p "8080:8080" \
    -e ARM_UID="1026" \
    -e ARM_GID="1001" \
    -v "/home/arm:/home/arm" \
    -v "/mnt/music/library:/home/arm/Music" \
    -v "/home/arm/logs:/home/arm/logs" \
    -v "/mnt/video/Movies:/home/arm/media" \
    -v "/etc/arm/config:/etc/arm/config" \
    --device="/dev/sr0:/dev/sr0" \
    --device="/dev/sr1:/dev/sr1" \
    --privileged \
    --restart "always" \
    --name "arm-rippers" \
    --cpuset-cpus='1,2,3' \
    automaticrippingmachine/automatic-ripping-machine:latest

monitor the container logs...

### Built from source files ?

- [ ] I Built the image from source

### What architecture are you seeing the problem on?

amd64

### Expected behavior?

I was the arm service to start monitoring the Bluray drive I have connnected, but it just fails.

### Relevant log output

```shell
[2023-04-23 18:00:02,373] DEBUG ARM: __init__.<module> Debugging pin: 12345
[2023-04-23 18:00:02,493] DEBUG ARM: utils.arm_alembic_get Alembic Head is: 95623e8c5d58
[2023-04-23 18:00:02,514] DEBUG ARM: utils.arm_db_get Database Head is: c3a3fa694636
[2023-04-23 18:00:02,514] INFO ARM: utils.arm_db_check Database is not current, update required. Head: 95623e8c5d58 DB: c3a3fa694636
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1900, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/default.py", line 736, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: no such table: ui_settings

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/arm/arm/runui.py", line 8, in <module>
    import arm.config.config as cfg  # noqa E402
  File "/opt/arm/arm/../arm/__init__.py", line 2, in <module>
    import arm.ripper
  File "/opt/arm/arm/../arm/ripper/__init__.py", line 3, in <module>
    from arm.ripper import logger, utils, makemkv, handbrake, identify  # noqa F401
  File "/opt/arm/arm/../arm/ripper/utils.py", line 24, in <module>
    from arm.ui import app, db
  File "/opt/arm/arm/../arm/ui/__init__.py", line 65, in <module>
    from arm.ui.database.database import route_database  # noqa: E402,F811
  File "/opt/arm/arm/../arm/ui/database/database.py", line 27, in <module>
    armui_cfg = ui_utils.arm_db_cfg()
  File "/opt/arm/arm/../arm/ui/utils.py", line 208, in arm_db_cfg
    armui_cfg = models.UISettings.query.get(1)
  File "<string>", line 2, in get
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/util/deprecations.py", line 402, in warned
    return fn(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/orm/query.py", line 947, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/orm/query.py", line 951, in _get_impl
    return self.session._get_impl(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/orm/session.py", line 2912, in _get_impl
    return db_load_fn(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/orm/loading.py", line 530, in load_on_pk_identity
    session.execute(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/orm/session.py", line 1712, in execute
    result = conn._execute_20(statement, params or {}, execution_options)
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1705, in _execute_20
    return meth(self, args_10style, kwargs_10style, execution_options)
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/sql/elements.py", line 333, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1572, in _execute_clauseelement
    ret = self._execute_context(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1943, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 2124, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1900, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/default.py", line 736, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: ui_settings
[SQL: SELECT ui_settings.id AS ui_settings_id, ui_settings.use_icons AS ui_settings_use_icons, ui_settings.save_remote_images AS ui_settings_save_remote_images, ui_settings.bootstrap_skin AS ui_settings_bootstrap_skin, ui_settings.language AS ui_settings_language, ui_settings.index_refresh AS ui_settings_index_refresh, ui_settings.database_limit AS ui_settings_database_limit 
FROM ui_settings 
WHERE ui_settings.id = ?]
[parameters: (1,)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
Starting web ui
[2023-04-23 18:00:04,321] DEBUG ARM: __init__.<module> Debugging pin: 12345
```

### Anything else?

I just made the switch today from a ubuntu install to a container install.
To do this, I removed the file from the udev folder, and I stopped the ui service in systemctl.

So, nothing else is catching the even when I insert a disk.

Thanks alot for all the hard work on this!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dbuttric** [comment]
I have resolved this issue.
I deleted the db dir from the arm user's home directory. That seems to have fixed this.

**Comment by microtechno9000** [comment]
This bug should be fixed in PR #807
